templates/CRM/Mailing/Form/InsertTokens.tpl

```diff
@@ -141,8 +141,10 @@ function selectValue( val, prefix) {
 
   cj.post( dataUrl, {tid: val}, function( data ) {
     var hide = (data.document_path && isPDF) ? false : true;
-    cj('.crm-html_email-accordion').toggle(hide);
+    cj('.crm-html_email-accordion, .crm-pdf-format-accordion').toggle(hide);
     cj('.crm-document-accordion').toggle(!hide);
+    cj('.crm-document-accordion .crm-accordion-header label').text(ts('Preview Document'));
```

Comment:
This is a `.tpl` file so needs to use `{/literal}{ts}` instead of `ts()`.

Also shouldn't the label get changed to 1 of 2 possible strings conditionally based on the variable `hide`?